src/util/system.h

```diff
@@ -16,449 +16,24 @@
 
 #include <compat/assumptions.h>
 #include <compat/compat.h>
-#include <logging.h>
-#include <sync.h>
-#include <util/fs.h>
-#include <util/settings.h>
-#include <util/time.h>
 
 #include <any>
-#include <map>
-#include <optional>
 #include <set>
 #include <stdint.h>
 #include <string>
-#include <utility>
-#include <vector>
-
-class ArgsManager;
```

Comment:
In commit "refactor: Extract common/args from util/system" (c02e451882bb1d220b944af3b444d4b529ac351a)

Would be good to drop the coment above line 7 "Server/client environment: argument handling, config file parsing"